azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java

```diff
@@ -67,51 +60,65 @@
 import java.nio.file.Paths;
 import java.security.KeyStore;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
+import java.util.Optional;
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 import java.util.concurrent.Future;
 import java.util.zip.ZipFile;
-import javax.sql.DataSource;
-import org.apache.commons.dbutils.QueryRunner;
-import org.apache.log4j.Logger;
+import org.apache.commons.lang.exception.ExceptionUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
- *  This class is responsible for launching a flow in a container.
+ *  This class is the entrypoint for launching a flow execution in a container.
+ *  It sets up the Azkaban Properties, the DAO, injects all the required classes, sets up the
+ *  execution directory along with the project files, creates the FlowRunner and submits it to
+ *  the executor service for execution.
+ *  It assumes that their is a certain directory structure consisting of all the dependencies.
+ *  The dependencies such as Hadoop, Hive, Pig, and other libraries.
  */
 public class FlowContainer {
 
   private static final String PROJECT_DIR = "project";
-  private static final String JOBTYPE_DIR = "jobtypes";
+  private static final String JOBTYPE_DIR = "plugins/jobtypes";
   private static final String CONF_ARG = "-conf";
   private static final String CONF_DIR = "conf";
+  private static final String FLOW_NUM_JOB_THREADS = "flow.num.job.threads";
+  private static final String DEFAULT_LOG_CHUNK_SIZE = "5MB";
+  private static final int DEFAULT_LOG_NUM_FILES = 4;
+  private static final int ARG_EXEC_ID = 0;
+  private static final int ARG_PROJECT_LOCATION = 1;
```

Comment:
Can you please rename it to EXEC_ID_INDEX and PROJECT_LOCATION_INDEX? Same for any other index. It is appropriate to denote that it is index for argument. You can even rename it to EXEC_ID_ARG_INDEX.